src/components/Details/SupplierDetails/SupplierDetails.js

```diff
@@ -136,9 +136,7 @@ function SupplierDetails(props) {
               <Grid item container justify="space-between">
                 <Typography className={classes.txtGreen}>Website: </Typography>
                 <Typography>
-                  {supplier.supplier_website !== null
-                    ? supplier.supplier_website
-                    : "N/A"}
+                  {supplier.supplier_website? <a href={supplier.supplier_website} target="_blank">Go to website</a> : "N/A"}
```

Comment:
An anchor tag with a blank target throws an error. You have to put a safety prop but i forgot what it's called, the console will tell you what to put.